packages/elements/src/pagination/index.ts

```diff
@@ -219,10 +260,14 @@ export class Pagination extends BasicElement {
    */
   private get totalPage (): number {
```

Comment:
You should just be able to use `internalMax` here and get rid of this getter.

Reply:
Already fixed